如何召开在线会议 (#8739)
xitu/gold-miner: article/ECMA-TC39/How-to-run-an-online-meeting.md

> * 原文作者：[Ecma TC39](https://github.com/tc39/how-we-work)
> * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
> * 本文永久链接：[https://github.com/xitu/gold-miner/blob/master/article/ECMA-TC39/How-to-run-an-online-meeting.md](https://github.com/xitu/gold-miner/blob/master/article/ECMA-TC39/How-to-run-an-online-meeting.md)
> * 译者：[PingHGao](https://github.com/PingHGao)
> * 校对者：[greycodee](https://github.com/greycodee)

> # 如何召开在线会议

Reviewer: ```suggestion

# 如何召开在线会议
```

改一下原文格式，换行

有时，可以通过举行额外的在线视频通话（例如通过 Zoom 或 Google Meet）来更有效地解决 TC39 中的问题，作为其他沟通形式的辅助手段。

## 你真的需要开会吗？

> 首先，想一想，你想解决什么问题？开会是解决问题的最佳方式吗？一些其他可能的技术可以避免额外的会议：

Reviewer 2: ```suggestion
首先，想一想，你想解决什么问题？开会是解决问题的最佳方式吗？下面这些工具也许可以避免额外的会议：
```
稍微意译一下

- GitHub 上的 issue/PR
- 电子邮件
- TC39 会议中的议程时段或分组讨论  
请记住，会议不仅需要您的时间来组织，还需要参加者的时间，包括所涉及的各项安排（而异步的基于文本的通信可以由参加者调整）。会议是一个重量级的工具，只有在真正有用时才使用。

如果这些其他沟通方式能很好地胜任，那太好了，开始沟通吧。在如下某些情况下，电话可能会有所帮助：
- 如果工作上很多往来沟通，书面交流最终比即时讨论更耗时。
- 如果您需要无法亲自参加 TC39 会议的人的帮助，并且在使用书面交流不切实际的情况下（例如，体量太大，每个人都被 GitHub 通知淹没）。
- 如果有很多事情要做，并且需要在合作者之间进行高层协调。

> 好的，如果您仍然认为会议是解决问题的最佳方式，请参阅下文，了解如何召开会议的一些建议。请注意，这些只是想法；随意组织和召开您的会议。

Reviewer 3: ```suggestion
好吧，如果您仍然认为会议是解决问题的最佳方式，请参阅下文，了解如何召开会议的一些建议。请注意，这些只是想法，当然你可以随意组织和召开您的会议。
```

## 安排会议

> [Doodle](https://doodle.com/) 是一种流行的工具，可让您创建快速投票来选择会议时间。使用 Doodle 的一些注意事项：

Reviewer 3: ```suggestion
[Doodle](https://doodle.com/) 是一款比较受欢迎的工具，它可让您创建快速投票来选择会议时间。使用 Doodle 的一些注意事项：
```

> - 不要包含太多或太少的选项 —— 太多填写起来很烦人，太少可能无法得到可接受的答案。 5-9 通常是一个很好的中间点。

Reviewer 3: ```suggestion
- 不要包含太多或太少的选项 —— 太多填写起来很烦人，太少可能无法得到可接受的答案。 5-9 个选项通常是一个很好的中间点。
```

> - 不要包含太多或太少的选项 —— 太多填写起来很烦人，太少可能无法得到可接受的答案。 5-9 个选项通常是一个很好的中间点。

Reviewer 2: ```suggestion
- 不要包含太多或太少的选项 —— 太多填写起来很烦人，太少可能无法得到可接受的答案。 5-9 个选项通常比较合适。
```
意译一下

- 不要包含您无法参加的时间/日期组合。
- 通常会议时间会长达一小时；有时半小时的时间不足以深入了解主题，而一个多小时通常会让与会者筋疲力尽。如果您不清楚如何将内容安排在一个小时内，请考虑真正重要的内容并将议程重点放在该部分。
- 如果与会者的时区不完全相同，请尝试选择对每个人都在正常工作时间内的合理的选项，并注意投票的默认时区（人们可能会默认在他们的时区！）。
- 在会议前两周发送 Doodle，并在会议前一周选择一个选项。
- 将 Doodle 发送给您特别想要邀请参加的每个人，并在包含所有可能想要参加的人的地方宣布。
- 在可能的情况下，了解与会者的任何特殊限制，例如，有些人可能更喜欢在其时区的正常工作时间之前或之后开会，或者更喜欢半小时左右的会议，并在这些限制范围内提供选项。

> Doodle 完成后，为会议创建日历邀请和议程并将其发送给与会者。

Reviewer 3: ```suggestion
Doodle 完成后，为会议创建日历邀请和议程并将其发送给参加会议的相关人员。
```

> ## 写议程

Reviewer 3: ```suggestion
## 编写议程
```

议程构成了会议的大纲并指导讨论。一个典型的议程可能包括：
- 会议的标题、日期和时间
- 与会者介绍（如果他们彼此不认识）

> - 问题的背景、动机、我们当前遇到的问题等

Reviewer 3: ```suggestion
- 问题的背景、动机、以及在哪里解决等等
```

- 在问题中需要解决的一些技术难点，以及详细背景和状态的链接
- 总结并确定下一步
  - 在会议期间，列出行动项目和负责人
  - 会议是否会再次召开以及召开的频率/持续时间等内容

在编写议程时，请记住：
- 不同的参会者会有不同的技术和组织背景；尽可能使会议让每个与会者都易于理解且感兴趣。
- 根据主题的需要，考虑保持技术和组织内容的平衡；有时保持内容为 100% 的技术或 100% 的组织会可能让部分与会人员没有参与感或难以设定下一步。

> [Google Docs](https://docs.google.com/) 通常是在议程上进行协作编辑的好工具。您可以使用他们的“全体可编辑，仅限链接共享”选项，让所有与会者根据您发送的初稿共同参与编写此议程。在会议期间，可以在议程中内嵌笔记。

Reviewer 3: ```suggestion
[Google Docs](https://docs.google.com/) 是一款在议程上进行协作编辑的好工具。您可以使用他们的“全体可编辑，仅限链接共享”选项，让所有与会者根据您发送的初稿共同参与编写此议程。在会议期间，可以在议程中内嵌笔记。
```

> [Google Docs](https://docs.google.com/) 是一款在议程上进行协作编辑的好工具。您可以使用他们的“全体可编辑，仅限链接共享”选项，让所有与会者根据您发送的初稿共同参与编写此议程。在会议期间，可以在议程中内嵌笔记。

Reviewer: ```suggestion
[Google Docs](https://docs.google.com/) 是一款在议程上进行协作编辑的好工具。你可以使用他们的“全体可编辑，仅限链接共享”选项，让所有与会者根据你发送的初稿共同参与编写此议程。在会议期间，你可以在议程中内嵌笔记。
```

## 创建日历事件

当 Doodle 投票完成后，我们将确定会议的时间和日期。为了帮助与会者准时参加，使用日历邀请很有帮助。 [Google 日历](https://calendar.google.com/) 在这方面很受欢迎。设置日历邀请的一些提示：

> - 在发布 Doodle 的地方发布会议，并指导人们与您联系以获取邀请。

Reviewer: ```suggestion
- 在发布 Doodle 的地方发布会议，并指导人们与你联系以获取邀请。
```

- 给会议拟一个标题和介绍，简要总结会议的目的，给出议程/笔记文档的链接。
- 鼓励受邀者编辑议程，并推荐其他相关的受邀者。
- 确保填写 Doodle 的每个人都会收到会议邀请。
- 有时与会者会有多个电子邮件地址，其中一些比其他地址能更好地处理 Google 日历邀请。在向电子邮件地址发送邀请时请记住这一点，如果需要，请向与会者询问信息。

> - 如果可能，将会议纳入 TC39 的委员会日历而不是您的个人日历。

Reviewer: ```suggestion
- 如果可能，将会议纳入 TC39 的委员会日历而不是你的个人日历。
```

- 在会议前一周发送此日历事件，并考虑在会议前一天或几个小时进行额外提醒。

## 举办会议

一些建议：
- 如有必要，请在开始时留一定时间等待参与者到达，但通常不超过 5 分钟。

> - 指定一个或多个笔记记录者，最好在会议前非正式地指定。如果有人擅长多任务处理，这可能是您，或者可能是与会者。为记录存放的位置（例如，内嵌在议程中）和会议期望的笔记风格（例如，仔细记录谁说了什么（如在 TC39 会议中），或只是高层次的观点）制定明确的指导方针。

Reviewer 2: ```suggestion
- 指定一个或多个会议记录者，最好在会议前非正式地指定。如果有人擅长多任务处理，这可能是您，或者可能是与会者。为记录存放的位置（例如，内嵌在议程中）和会议期望的笔记风格（例如，仔细记录谁说了什么（如在 TC39 会议中），或只是高层次的观点）制定明确的指导方针。
```

- 介绍自己及会议目的；如果需要，鼓励每个人做自我介绍。
- 控制时间：如果某些主题讨论花费的时间太长，并且议程上还有其他重要主题需要在这次特定会议上进行讨论，则将其结束。

> - 作为会议主持人，不要说得太少或太多：确保充分介绍主题并提供足够的上下文，之后交出剩余的时间，以便与会者可以分享他们的见解。主持会议，以便每个人都能发表自己的观点。

Reviewer 2: ```suggestion
- 作为会议主持人，发言内容要适当，不宜过少或过多；确保充分介绍主题并提供足够的上下文，之后交出剩余的时间，以便与会者可以分享他们的见解。主持会议，以便每个人都能发表自己的观点。
```

## 会议之后的行动

> - 根据与会人员的一致意见，将笔记张贴在适当的地方，以便对主题感兴趣的每个人都可以阅读。这可能在公共的 GitHub 存储库中（对于大多数会议），也可能在一个单独的私人地方。确保向对该主题感兴趣的人宣传这些笔记。您可以使用 [此脚本](https://lifehacker.com/this-script-converts-google-documents-to-markdown-for-e-511746113) 将 Google 文档转换为 Markdown 文件。

Reviewer: ```suggestion
- 根据与会人员的一致意见，你可以将笔记张贴在适当的地方，以便对主题感兴趣的每个人都可以阅读，比如说放在公共的 GitHub 存储库中（对于大多数会议），也可能在一个单独的私人的地方。你还需要确保向对该主题感兴趣的人宣传这些笔记。你可以使用 [该脚本](https://lifehacker.com/this-script-converts-google-documents-to-markdown-for-e-511746113) 将 Google 文档转换为 Markdown 文件。
```

> - 如果小组决定将其设为定期会议，请将原始 Google 日历活动设置为定期活动 —— 这比每次都创建一个新活动要容易得多。如果您编辑说明或添加受邀者，请注意是仅用于一次会议还是重复的应用 —— 很容易不小心将人员添加到仅一次会议！通常，重复使用相同的议程/笔记文档效果很好，并且只需在同一文档的开头为下一次会议添加议程即可。

Reviewer: ```suggestion
- 如果小组决定将其设为定期会议，请将原始 Google 日历活动设置为定期活动 —— 这比每次都创建一个新活动要容易得多。如果你编辑说明或添加受邀者，请注意是仅用于一次会议还是针对定期活动的应用 —— 很容易不小心将人员添加到仅一次会议！通常，定期会议使用相同的议程/笔记文档的话效果很好，并且只需在同一文档的开头为下一次会议添加议程即可。
```

> - 如果某些与会者采取了行动项目，但您没有看到该项目的进展，请考虑单独向这些人发送电子邮件以询问状态，而不是在未来的会议中公开询问。

Reviewer: ```suggestion
- 如果某些与会者采取了行动项目，但你没有看到该项目的进展，请考虑单独向这些人发送电子邮件以询问状态，而不是在未来的会议中公开询问。
```

- 会议的标题、日期和时间
- 与会者介绍（如果他们彼此不认识）
- 问题的背景、动机、我们当前遇到的难题等
- 在问题中需要解决的一些技术难点，以及详细背景和状态的链接
- 总结并确定下一步
  - 在会议期间，列出行动项目和负责人
  - 会议是否会再次召开以及召开的频率/持续时间等内容

在编写议程时，请记住：
- 不同的参会者会有不同的技术和组织背景；尽可能使会议让每个与会者都易于理解且感兴趣。
- 根据主题的需要，考虑保持技术和组织内容的平衡；有时保持内容为 100% 的技术或 100% 的组织会可能让部分与会人员没有参与感或难以设定下一步。

[Google Docs](https://docs.google.com/) 是一款在议程上进行协作编辑的好工具。你可以使用他们的“全体可编辑，仅限链接共享”选项，让所有与会者根据你发送的初稿共同参与编写此议程。在会议期间，你可以在议程中内嵌笔记。

## 创建日历事件

当 Doodle 投票完成后，我们将确定会议的时间和日期。为了帮助与会者准时参加，使用日历邀请很有帮助。 [Google 日历](https://calendar.google.com/) 在这方面很受欢迎。设置日历邀请的一些提示：
- 在发布 Doodle 的地方发布会议，并指导人们与你联系以获取邀请。
- 给会议拟一个标题和介绍，简要总结会议的目的，给出议程/笔记文档的链接。
- 鼓励受邀者编辑议程，并推荐其他相关的受邀者。
- 确保填写 Doodle 的每个人都会收到会议邀请。
- 有时与会者会有多个电子邮件地址，其中一些比其他地址能更好地处理 Google 日历邀请。在向电子邮件地址发送邀请时请记住这一点，如果需要，请向与会者询问信息。
- 如果可能，将会议纳入 TC39 的委员会日历而不是你的个人日历。
- 在会议前一周发送此日历事件，并考虑在会议前一天或几个小时进行额外提醒。

## 举办会议

一些建议：
- 如有必要，请在开始时留一定时间等待参与者到达，但通常不超过 5 分钟。
- 指定一个或多个会议记录者，最好在会议前非正式地指定。如果有人擅长多任务处理，这可能是您，或者可能是与会者。为记录存放的位置（例如，内嵌在议程中）和会议期望的笔记风格（例如，仔细记录谁说了什么（如在 TC39 会议中），或只是高层次的观点）制定明确的指导方针。
- 介绍自己及会议目的；如果需要，鼓励每个人做自我介绍。
- 控制时间：如果某些主题讨论花费的时间太长，并且议程上还有其他重要主题需要在这次特定会议上进行讨论，则将其结束。
- 作为会议主持人，发言内容要适当，不宜过少或过多；确保充分介绍主题并提供足够的上下文，之后交出剩余的时间，以便与会者可以分享他们的见解。主持会议，以便每个人都能发表自己的观点。

## 会议之后的行动

> - 根据与会人员的一致意见，你可以将笔记张贴在适当的地方，以便对主题感兴趣的每个人都可以阅读，比如说放在公共的 GitHub 存储库中（对于大多数会议），也可能在一个单独的私人的地方。你还需要确保向对该主题感兴趣的人宣传这些笔记。你可以使用 [该脚本](https://lifehacker.com/this-script-converts-google-documents-to-markdown-for-e-511746113) 将 Google 文档转换为 Markdown 文件。

Author: ```suggestion
- 根据与会人员的一致意见，你可以将笔记张贴在适当的地方，以便对主题感兴趣的每个人都可以阅读。比如说放在公共的 GitHub 存储库中（对于大多数会议），或者一个单独的私人的地方。你还需要确保向对该主题感兴趣的人宣传这些笔记。你可以使用[该脚本](https://lifehacker.com/this-script-converts-google-documents-to-markdown-for-e-511746113) 将 Google 文档转换为 Markdown 文件。
```
